modular_chomp/code/modules/emotes/definitions/audiable.dm

```diff
@@ -67,6 +67,14 @@
 	emote_message_3p_target = "gwahs at TARGET."
 	emote_sound = 'modular_chomp/sound/voice/emotes/gwah.ogg'
 
+/decl/emote/audible/caw
+	key = "caw"
+	emote_message_1p = "You caw!"
+	emote_message_3p = "caws!"
+	emote_message_1p_target = "You caw at TARGET."
+	emote_message_3p_target = "caws at TARGET."
+	emote_sound = 'modular_chomp/sound/voice/emotes/caw.ogg' // Copyright Sampling+ 1.0 Vixxus (freesound.org) for the source audio.
```

Comment:
are you sure that name is correct? Vixxus

Reply:
In my mind it 100% was until I read your comment, I even was about to go "Uhm, actually yeah" then double checked and went: 'v' "oh".

Sorry for making you guys check so many times, I appreciate your patience and will try to make a better work next time uvu